src/Raven.Client/Documents/Session/AsyncSessionDocumentTimeSeries.cs

```diff
@@ -138,6 +138,8 @@ await ServeFromCacheOrGetMissingPartsFromServerAndMerge(timeseries, from, to, ra
                 DateTime from,
                 DateTime to,
                 List<TimeSeriesRangeResult> ranges,
+                int? skip,
+                int? take,
```

Comment:
Check that the naming is consistent.
` GetDocumentsCommand(int start, int pageSize)`